.snyk

```diff
@@ -5,10 +5,11 @@ ignore:
   'snyk:lic:golang:github.com:hashicorp:hcl:v2:MPL-2.0':
     - '*':
         reason: This license is addressed by including acknowledgments in each release
+        expires: 2022-09-03T08:14:03.074Z
```

Comment:
I think not having an expires here was intentional https://github.com/snyk/snyk-iac-rules/commit/9677e34b5d6e9f9d836b6b671bd7d0679c40d33d